build.py

```diff
@@ -0,0 +1,64 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPL-3.0 2016-present Scille SAS
+
+import os
+import sys
+import logging
+import subprocess
+
+YELLOW_FG = "\x1b[33m"
+DEFAULT_FG = "\x1b[39m"
+
+log = logging.getLogger("build")
+logging.basicConfig(
+    level=logging.DEBUG, format=f"{YELLOW_FG}[%(levelname)s] %(message)s{DEFAULT_FG}"
+)
+
+PYTHON_PATH = sys.executable
+log.debug(f"PYTHON_PATH={PYTHON_PATH}")
+
+
+def run(cmd, **kwargs):
+    log.debug(f">>> {cmd}")
+    ret = subprocess.run(cmd, shell=True, check=True, **kwargs)
+    return ret
+
+
+def in_cibuildwheel():
+    return bool(int(os.environ.get("CIBUILDWHEEL", "0")))
+
+
+def check_venv():
+    # Both conda and venv are configured, pick conda
+    if os.environ.get("CONDA_PREFIX") and os.environ.get("VIRTUAL_ENV"):
+        del os.environ["VIRTUAL_ENV"]
+    # Build with cibuildwheel detected, set `VIRTUAL_ENV`
+    if in_cibuildwheel():
+        os.environ["VIRTUAL_ENV"] = os.path.dirname(os.path.dirname(PYTHON_PATH))
+        log.debug(
+            f"Build with cibuildwheel detected, set `VIRTUAL_ENV` as: {os.environ['VIRTUAL_ENV']}"
+        )
+    if not os.environ.get("CONDA_PREFIX") and not os.environ.get("VIRTUAL_ENV"):
+        raise RuntimeError("A virtual env is required to run `maturin develop`")
+
+
+def build():
+    # Debug
+    log.debug("Environment:")
+    for key, value in os.environ.items():
+        log.debug(f"- {key} = {value}")
+
+    run(f"{PYTHON_PATH} --version")
+    run(f"{PYTHON_PATH} -m pip freeze")
+    run(f"{PYTHON_PATH} misc/generate_pyqt.py")
+    check_venv()
+    release = "--release" if in_cibuildwheel() else ""
```

Comment:
It would be useful to be able to trigger release builds even with `poetry install`, for instance using `FORCE_MATURIN_RELEASE=1 poetry install`:
```suggestion
    release = "--release" if in_cibuildwheel() or force_maturin_release() else ""
```
with `force_maturin_release` defined as:
```python
def force_maturin_release():
    return bool(int(os.environ.get("FORCE_MATURIN_RELEASE", "0")))
```

Reply:
done in ed05818